src/util/emitUtils.js

```diff
@@ -576,11 +577,14 @@ function doEmit(nsp, key, obj) {
   if (!toggle.isFeatureEnabled('enableClientStats')) {
     nsp.emit(key, newObjectAsString);
```

Comment:
The toggle is only for the client stats - need to track emit here too.

Reply:
fixed